src/mlpack/methods/edge_boxes/feature_parameters.hpp

```diff
@@ -0,0 +1,128 @@
+/**
+ * @file feature_parameters.hpp
+ * @author Nilay Jain
+ *
+ * Implementation of feature parameter class.
+ */
+
+#ifndef MLPACK_METHODS_EDGE_BOXES_FEATURE_PARAMETERS_HPP
+#define MLPACK_METHODS_EDGE_BOXES_FEATURE_PARAMETERS_HPP
+
+namespace mlpack {
+namespace structured_tree {
+ 
+//! This class holds all the fields for the FeatureExtraction class.
+class FeatureParameters
+{
+ public:
+
+  //! getter and setter methods for all the fields in class.
+  void NumImages(size_t value) { numImages = value; }
+  size_t NumImages() const { return numImages; }
+
+  void RowSize(size_t value) { rowSize = value; }
```

Comment:
Can you add a comment for each function? You could use the comments from ``edge_boxes_main.cpp``.